python-avd/pyavd/_eos_designs/structured_config/overlay/router_path_selection.py

```diff
@@ -149,44 +135,48 @@ def _get_path_group_id(self: AvdStructuredConfigOverlayProtocol, path_group_name
             return config_id
         return 500
 
-    def _get_local_interfaces_for_path_group(self: AvdStructuredConfigOverlayProtocol, path_group_name: str) -> list:
+    def _set_local_interfaces_for_path_group(
+        self: AvdStructuredConfigOverlayProtocol, path_group_name: str, path_group_item: EosCliConfigGen.RouterPathSelection.PathGroupsItem
+    ) -> None:
         """
         Generate the router_path_selection.local_interfaces list.
 
         For AUTOVPN clients, configure the stun server profiles as appropriate
         """
-        local_interfaces = []
         if path_group_name not in self.shared_utils.wan_local_path_groups:
```

Comment:
Once your rename the `path_group_item` to `path_group`, use:
```suggestion
        if path_group.name not in self.shared_utils.wan_local_path_groups:
```

Reply:
Fixed.